.golangci.yml

```diff
@@ -0,0 +1,109 @@
+---
+run:
+  concurrency: 6
+  deadline: 5m
+  skip-files:
+    - ".*_test\\.go"
```

Comment:
we definitely want to exclude unit test files from linting? Just double checking.

Reply:
yep, but as with the other "ignored" linters in this file, we might want, at some point, remove this and fix everything. Right now we need to get this in and fix code (not tests) to have this run and check on every PR w/o asking ppl to gofmt as you noticed.

Reply:
ok sounds good